manual/Tasks/junitlauncher.html

```diff
@@ -366,6 +366,13 @@ <h5>Test result formatter</h5>
         </td>
         <td>No</td>
     </tr>
+    <tr>
+        <td>extension</td>
+        <td>Extension to append to the output filename.
+            <p><em>Since Ant 1.10.13</em></p>
+        </td>
+        <td>No; defaults to <q>txt</q></td>
```

Comment:
Hello @azotcsit, I think the "defaults to txt" part will be misleading since for the legacy-xml formatter it will rightly default to `xml`. Perhaps, change this part to "defaults to `xml` for the `legacy-xml` formatter and to `txt` for the rest"?

Reply:
Sounds reasonable, I've changed it. Thanks @jaikiran!